subsys/logging/log_backend_spinel.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log_backend.h>
+#include <logging/log_output.h>
+#include "log_backend_std.h"
+
+#ifndef CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE
+#define CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE 0
+#endif
+
+static u8_t char_buf[CONFIG_LOG_BACKEND_SPINEL_BUFFER_SIZE];
+static bool panic_mode;
```

Comment:
what is this needed for? I found only write to this but no read.

Reply:
I think that I was unnecessarily suggested by another logging backend and I forgot to remove it. Fixed.